src/vivarium_inputs/validation/sim.py

```
@@ -280,8 +280,7 @@ def _validate_relative_risk(data: pd.DataFrame, entity: Union[RiskFactor, Covera
         raise NotImplementedError()
 
     validation_utilities.check_value_columns_boundary(data, boundary_value=VALID_RELATIVE_RISK_RANGE[0],
-                                                      boundary_type='lower', value_columns=['value'],
-                                                      error=DataTransformationError)
+                                                      boundary_type='lower', value_columns=['value'])
```

Wait I think we still want to throw an error here in sim validators? If RR isn't meeting the lower boundary in sim validators, do we really just want to warn and carry on?

---

on Thursday coworking, we decided to give warning, (I don't know whether we want to do it temporarily or forever thou) We probably revisit it since some risk has low boundary a little off from 1 while the other is very off from 1

---

I think we want a soft rr boundary at 1 (some risks legitimately have protective effects) and a hard rr boundary at 0 (non-inclusive) since RR <=0 doesn't make sense.